views/authors.erb

```diff
@@ -1,19 +1,59 @@
-<h1>Authors</h1>
-
-<%= erb :'components/pagination', locals: {label: 'Top pagination', list: list} %>
-<ul>
-  <% list.items.each do |result| %>
-    <li>
-      <%= result.num_matches %> | Author: <%=result.author%><% if result.heading_link? %> | <a href="<%= result.heading_link%>">Heading Information</a><%end%>
-      <% if result.has_cross_references?  %>
-        <br/> 
-        <% result.cross_references.each do | cr | %>
-          <% if cr.kind == 'see' %> See: <% end %>
-          <a href="<%=cr.url%>"><%= cr.author%></a> (<%= cr.num_matches %>) <% if cr.heading_link? %> <a href="<%= cr.heading_link %>">Heading Information</a> <% end %>
+<% if list.show_table? %>
+  <table class="browse-table" aria-labelledby="maincontent">
+    <thead>
+      <tr>
+        <th>Records</th>
+        <th>Author</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% list.items.each do |result| %>
+        <% if result.match_notice? %>
+          <tr class="match-<%= list.num_matches == 0 ? 'none' : 'notice' %>">
+            <td colspan="2">
+              <%= list.match_text %>
+            </td>
+          </tr>
+        <% else %>
+          <tr <% if result.exact_match? %>class="exact-match"<% end %>>
+            <td>
+              <%= result.record_text %>
+            </td>
+            <td>
+              <dl>
+                <dt class="visually-hidden">Author:</dt>
+                <dd>
+                  <% if result.num_matches > 0 %><a href="<%= result.url%>"><% end %>
+                    <span <% if result.exact_match? %>class="strong"<% end %>>
+                      <%=result.author%>
+                    </span>
+                  <% if result.num_matches > 0 %></a><% end %>
+                  <% if result.heading_link? %>
+                    <a href="<%= result.heading_link%>" class="vernacular pipe">View Heading Information</a>
+                  <% end %>
+                </dd>
+                <% if result.has_cross_references? %>
+                  <% result.cross_references.each do | cr | %>
+                    <% if cr.kind == 'see' %>
+                      <dt>See:</dt>
+                    <% end %>
+                    <dd>
+                      <% if cr.num_matches > 0 %><a href="<%= cr.url%>"><% end %>
+                        <%=cr.author%>
+                      <% if cr.num_matches > 0 %>
+                        </a> (<%= cr.record_text %>)
+                      <% end %>
+                      <% if cr.heading_link? %>
+                        <a href="<%= cr.heading_link %>" class="vernacular pipe">View Heading Information</a>
+                      <% end %>
+                    </dd>
```

Comment:
There's probably a way to do this better, so there aren't as many if statements in the view, but we don't need to fix it now especially since how this will work/look in the end is pretty uncertain.